src/header/index.js

```diff
@@ -0,0 +1,19 @@
+import styles from './styles.css'
+import logo from '../../images/logo.png'
+import { EditableText } from '../editableText';
+
+export const Header = () => {
+  const links = ['Home', 'Task list', 'Contacts']
+
+  return (
+    <header className={styles.header}>
+      <img src={logo} />
+      <nav>
+        <ul className={styles.list}>
+          {links.map((link, index) => <li key={index} ><a href="/" className={styles[`${link.toLowerCase().replace(/\s+/, '-')}`]} >{link}</a></li>)}
```

Comment:
`styles[`${link.toLowerCase().replace(/\s+/, '-')}`]` why do you need it?

Reply:
In order to make background-position I needed different classes. And .toLowerCase().replace(/\s+/, '-') is needed so that the classes are not capitalized and there is no space in the Task list, I replaced it with -

Reply:
Then everything will break. I'll figure it out)